pkg/policy/resolve.go

```diff
@@ -73,6 +75,8 @@ type policyContext struct {
 	defaultDenyIngress bool
 	defaultDenyEgress  bool
 
+	orgin ruleOrigin
```

Comment:
```suggestion
	origin ruleOrigin
```

the linter is also nagging about it.

Reply:
:facepalm: